.github/workflows/configure_bazelrc.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
 
+touch "$HOME/.bazelrc"
+chmod 0600 "$HOME/.bazelrc"
+
+# Configure flags for bzlmod
+echo "common:bzlmod --enable_bzlmod" >> "$HOME/.bazelrc"
+echo "common:workspace --noenable_bzlmod" >> "$HOME/.bazelrc"
+
 if [ "${BUILDBUDDY_API_KEY:-}" ]; then
-  touch "$HOME/.bazelrc"
-  chmod 0600 "$HOME/.bazelrc"
   echo "build --bes_backend=grpcs://remote.buildbuddy.io"
   echo "build --bes_results_url=https://app.buildbuddy.io/invocation/"
   echo "build --remote_cache=grpcs://remote.buildbuddy.io"
-  echo "build --remote_header=x-buildbuddy-api-key=$BUILDBUDDY_API_KEY" > "$HOME/.bazelrc"
+  echo "build --remote_header=x-buildbuddy-api-key=$BUILDBUDDY_API_KEY" >> "$HOME/.bazelrc"
 fi
+
```

Comment:
@f0rmiga Could you give me a hand understanding why this isn't getting picked up by the workflows? Locally, the file in $HOME/.bazerlc gets created correctly, and as far as I understand this script does run.

Reply:
If you could revert the changes you made to this file, I'll get rid of it entirely.